src/index.js

```diff
@@ -2,10 +2,20 @@
 
 const debug = require('debug')
 const log = debug('libp2p:websockets')
-const SWS = require('simple-websocket')
+const SW = require('simple-websocket')
+const isNode = require('detect-node')
+let SWS
+if (isNode) {
+  SWS = require('simple-websocket-server')
```

Comment:
this still requires to have a fake `simple-websocket-server` in the webpack config :/ not sure how to better solve it though